src/app/loadout-builder/filter/ModPicker.tsx

```diff
@@ -243,23 +236,18 @@ function ModPicker({
     _.groupBy(queryFilteredMods, (mod) => mod.plug.plugCategoryHash)
   ).sort(sortModGroups);
 
-  const plugCategoryHashOrder = groupedMods.map((mods) => mods[0].plug.plugCategoryHash);
-
   const autoFocus =
     !isPhonePortrait && !(/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream);
 
-  const footer = Object.values(lockedModsInternal).some((f) => Boolean(f?.length))
```

Comment:
I got rid of the boolean condition here as the footer popping in and out when choosing the first mod has always annoyed me.